api-docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Tock API documentation
+
+## Before you start
+You'll need to have an API key, and you'll need to have it added to the `GSA_TE_Tock_Library` script. Contact a Tock developer or ping the `#tock-dev` slack channel for help.
+
+
+## Table of contents
+
+- [Hours by quarter by user](hours-by-quarter-by-user.md): Fetches an hourly summary of all submitted timecards by year, by quarter, and by user.
+- [Hours by quarter](hours-by-quarter.md): Fetches an hourly summary, by year and quarter, of all submitted timecards.
+- [Project info](project-info.md): Fetches information about a specific project.
+- [Projects](projects.md): Fetches information about various projects.
+- [Reporting period (audit specific time period)](reporting-period-audit-specific.md): Fetches a list of all users who have not submitted a timecard for the specified period.
+- [Reporting periods](reporting-period-audit.md): Fetches a list of all available reporting periods and basic information about them.
+- [Timecards](timecards.md): Fetches a list of all submitted timecards and related information.
+- [User data](user-data.md): Fetches a list of all users, along with organizational information for each.
+- [Users](users.md): Fetches a list of all users, along with basic information about them.
+
+--------
+
+The Tock API documentation is written in
+[GitHub-flavored markdown][gh-md] and is best read using the GitHub interface.
+
+[gh-md]: https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
```

Comment:
Where is this documentation coming from? Is it tangential to this work? It’s okay if it is. I’m just unclear on what it’s doing in the pr.

Reply:
I put it here to keep it parallel with the dev docs where it was already noted, but I don't have strong opinions about it one way or the other.